src/core/i18n/es/translation.json

```diff
@@ -149,5 +149,6 @@
   "submission_error_title": "Error en el envío",
   "submission_success": "Gracias por completar la encuesta del Diario de Uso del Tiempo. Sus respuestas han sido registradas.",
   "submission_success_title": "Su encuesta se ha enviado correctamente!",
-  "nighttime_activity_title": "Actividad Nocturna"
+  "nighttime_activity_title": "Actividad Nocturna",
+  "error_invalid_url": "An error occurred while loading the Time Use Diary. Please try again later."
```

Comment:
Remind me again - the end user is responsible for maintaining translations, correct?

Reply:
Correct. I will ask them to add the correct translation for that line